scripts/vscode-build.ps1

```diff
@@ -1,6 +1,7 @@
 param (
   [Parameter(Mandatory = $true)][string]$filePath,
-  [string]$msbuildEngine = "vs"
+  [string]$msbuildEngine = "vs",
+  [string]$framework = $null
```

Comment:
Use `""` instead of `$null` here. In powershell `$null` will be converted to `""` when assigned to a `[string]` target. Example

```
P:\devops-util\triage> [string]$a = $null
P:\devops-util\triage> $a -eq $null
False
P:\devops-util\triage> $a -eq ""
True
```